Languages/Android/questions.json

```diff
@@ -550,6 +550,33 @@
       "trueOrFalse": false,
       "shortAns": "It attaches and detaches fragments from their host activity.",
       "tag": "fragment"
+    },
+    {
+      "id": 61,
+      "question": "To pass an object to different activities, which interface should the object implement?",
+      "details": "To pass an object to different activities, the object should implement either Serializable or Parcelable interface.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "It should implement either serializable or parcelable.",
+      "tag": "intent"
+    },
+    {
+      "id": 62,
+      "question": "What is a dangerous permission?",
+      "details": "Permissions that are dangerous include those that affect user data in some way. For example reading or writing data from device's storage",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "It is a permission that may affect user's privacy.",
```

Comment:
Again please don't use the word in the shortAns. 

"what does a cat eat?" 
a) oranges
b) tacos
c) a cat eats mice
d) trees. 

making the answer so obvious defeats the point of the app.